### DISCUSSION TITLE
Tailwind ignores class changes in html

### DISCUSSION THREAD

**Discussion by reposman33** [tailwindlabs/tailwindcss#8368] [Help]
Hello,

Tailwindcss ignores changes I make in the class attribute of a html element.

To reproduce I

- installed Tailwindcss in a simple Angular app using the steps in [https://tailwindcss.com/docs/guides/angular](https://tailwindcss.com/docs/guides/angular)
- I run `yarn start` (which calls `ng serve`)
- the application runs and in a component template I add a class attribute : 

```
<div class="bg-slate-100 shadow-xl">
...
</div>
```
- I save the template and the div is being displayed with a nice slate background and a large box shadow.
- I change the class attribute to 

```
<div class="bg-slate-50 shadow-xl">
...
</div>
```

- I save the template and the div is being displayed with an all white (not slate) background and a large box shadow.
- When I look in the developers tool Inspector tab I _only_ see the `shadow-xl` class defined in the right pane. The `bg-slate-50` class is listed on the _element_ but the _definition_ is missing in the right pane.

Further changes to the class attribute (eg changing it to `bg-indigo-100`) are ignored.

Looks like only the initial build (ng serve) compiles Tailwindcss but subsequent builds ignore new Tailwindcss classes.

What do I need to add here?

versions involved:

Angular: "@angular/core": "~12.2.0",
Tailwindcss:  "tailwindcss": "^3.0.24",
"postcss": "^8.4.13",
 "autoprefixer": "^10.4.7",

**Comment by reposman33** [comment]
For future reference: it works when upgrading Angular and Typescript.

    "@angular/core": "~13.3.5",
    "autoprefixer": "^10.4.7",
    "postcss": "^8.4.13",
    "tailwindcss": "^3.0.24",
    "typescript": "^4.4.2"

**Reply by arnab8820** [reply]
Hi, I am also struggling with the same issue. My angular version is already 13. Could you share some more info on how you fixed it?
